packages/html/src/spreadsheet/tests/spreadsheet-print.tsx

```diff
@@ -26,13 +26,14 @@ export default () =>(
                         <Button themeColor="primary">Save</Button>
                     </>
                 }>
-                    <TabStrip header={null} position="top">
-                        <TabStripItemsWrapper orientation="horizontal">
-                            <TabStripItems>
+                    <TabStripNormal header={null} position="top"
```

Comment:
Do we need this position="top"? Isn't TabStripNormal a TabStrip with position top?